includes/Core/Util/Input.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Input abstraction class.
```

Comment:
SImilar to other class files, this doc should just be "Class x", i.e. "Class Google\Site_Kit\Core\Util\Input".

Reply:
There is a bit of inconsistency around this in the codebase 😄

Reply:
Let's fix that at some point. :)